src/Squirrel/DeltaPackage.cs

```diff
@@ -267,8 +263,9 @@ void applyDiffToFile(string deltaPath, string relativeFilePath, string workingDi
 
                 if (relativeFilePath.EndsWith(".hdiffz", StringComparison.InvariantCultureIgnoreCase)) {
                     this.Log().Info("Applying HDiffz to {0}", relativeFilePath);
+                    // "-f" is a special option that allows hpatchz to overwrite an already created temp file.
```

Comment:
It might also be useful to mention why do we need the switch.

Reply:
Ok, I rewrote it te be more clear.